openshift/console
# OCPBUGS-9447: Helm chart Nodejs Ex K could not be installed via Form view

**Fixes**: 

https://issues.redhat.com/browse/OCPBUGSM-47478

**Analysis / Root cause**: 

Helm Chart "Nodejs Ex K" could not be installed via form mode.

**Solution Description**: 

The `formData` was being passed into a `Prune` function, which was used to remove all the empty and null objects and their keys. This caused the issue since for the form to be successfully executed, some of these deleted keys were required.

As the solution, the call to the `Prune` function is cancelled.

**Screen shots / Gifs for design review**: 


https://user-images.githubusercontent.com/47265560/190225629-d8852d8c-ecb1-46ce-9114-28d27fb72c66.mp4

**Unit test coverage report**: 

No Changes

**Test setup:**

1. Switch to Developer perspective
2. Navigate to Add > Helm Chart (catalog)
3. Search for nodejs and select "Nodejs Ex K"
4. Click on "Install Helm Chart"
5. Ensure the form view is opened (as yaml works fine)
6. Press Install


**Browser conformance**:
